demo/b2share_demo/cli.py

```diff
@@ -37,22 +52,161 @@ def demo():
 @click.option('-v', '--verbose', is_flag=True, default=False)
 def load(verbose):
     """Load demonstration data."""
-    _create_records()
+    with db.session.begin_nested():
+        communities = _create_communities(verbose)
+        block_schemas = _create_block_schemas(communities)
```

Comment:
Good catch, I'll have to remove the assignment.
I plan to split this in multiple commands later. They won't depend on other functions' return value but access the database instead.